packages/libs/error-stack/src/report.rs

```diff
@@ -352,13 +440,10 @@ impl<C> Report<C> {
     #[cfg(feature = "spantrace")]
     pub fn span_trace(&self) -> Option<&SpanTrace> {
         #[cfg(not(nightly))]
-        let span_trace = self.inner.span_trace.as_ref()?;
+        return None;
```

Comment:
Spantraces are not tight to the nightly channel. As they are an attachment, you can get them by iterating the frames. For nightly channels we should keep the `request_ref` as done below.